cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -791,97 +604,10 @@ def _fail_task(self, task, reason):
-            stream_body(entity, boundary, param, task, ioctx)
-        except cherrypy.HTTPError as e:
-            raise cherrypy.HTTPError(e.status, e._message)
-        except Exception as e:
-            if str(e).lstrip() != 'timed out':
-                shell.run('rbd rm %s' % task.tmpPath)
-                self._fail_task(task, str(e))
-            if param.slice_offset == 0:
-                shell.run('rbd rm %s' % task.tmpPath)
-
+            UploadHandler(req, self.upload_tasks).handle_upload()
+        except Exception, e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)